### ISSUE TITLE
how to refer to texlive buildEnv ?

### ISSUE THREAD

**Issue by teto** [rycee/home-manager#223]
I am trying to use latex on nixos and saw the hm module hence I did:
```
  programs.texlive = {
    enable=true;
    extraPackages = tpkgs: {
      inherit (tpkgs) collection-fontsrecommended algorithms cleveref latexmk;
    };
  };
```
Problem is to use vimtex in neovim, I need to set the path to latexmk and bibtex. If I use the programs stuff, how can I retrieve a reference to the env ?

I ended up with sthg like
```
  texliveEnv = pkgs.texlive.combine {
    # tabularx is not available
    inherit (pkgs.texlive) scheme-small cleveref latexmk bibtex algorithms cm-super
    csvsimple subfigure  glossaries;
   };
  home.file.".latexmk".text =  ''
    $bibtexviewer=${texliveEnv}/bin/bibtex
    '';
... and writing in neovim init.vim
let g:vimtex_compiler_latexmk.executable='${myTexliveEnv}/bin/latexmk'
```

**Comment by rycee** [comment]
Good point :-)

This should be fixed in 46a94cce566a5293ae3fb8876ec3fd916eda9fa8, if you update to latest master than you should be able to get the built package from `config.programs.texlive.package`. E.g.

```
home.file.".latexmk".text =  ''
  $bibtexviewer=${config.programs.texlive.package}/bin/bibtex
'';
```

**Comment by teto** [comment]
thanks for the fix.
